### ISSUE TITLE
make the status of multiple attached sessions surveyable

### ISSUE THREAD

**Issue by slingamn** [oragono/oragono#475]
1. Clients should be able to see a list of their attached sessions (time created, time active, IP, host) via NickServ
1. Same for operators

**Comment by slingamn** [comment]
Does ZNC (or any other bouncer) have an existing feature we can model this after?

**Comment by DanielOaks** [comment]
Yeah ZNC has `LISTCLIENTS`:

```
[07:11:01]  <dan->	listclients
[07:11:01]  <*status>	+---------------+----------+------------+
[07:11:01]  <*status>	| Host          | Network  | Identifier |
[07:11:01]  <*status>	+---------------+----------+------------+
[07:11:01]  <*status>	| 192.168.0.201 | freenode |            |
[07:11:01]  <*status>	| 192.168.0.90  | freenode |            |
[07:11:01]  <*status>	| 192.168.0.201 | iiii     |            |
[07:11:01]  <*status>	| 192.168.0.90  | iiiii     |            |
[07:11:01]  <*status>	| 192.168.0.201 | iiii2    |            |
[07:11:01]  <*status>	| 192.168.0.90  | iiii2    |            |
[07:11:01]  <*status>	| 192.168.0.201 | sssssss  |            |
[07:11:01]  <*status>	| 192.168.0.90  | sssssss  |            |
[07:11:01]  <*status>	| 192.168.0.201 | pppppp   |            |
[07:11:01]  <*status>	| 192.168.0.201 | uuuuuuuu |            |
[07:11:01]  <*status>	+---------------+----------+------------+
```
(each of those entries are network names, etc)

**Comment by slingamn** [comment]
Implemented in #477
